src/components/UserCollective.js

```diff
@@ -74,43 +74,45 @@ class UserCollective extends React.Component {
       },
       'organization.collective.edit': {
         id: 'organization.collective.edit',
-        defaultMessage: 'edit organization',
+        defaultMessage: 'edit Organization',
       },
       'user.collective.edit': {
         id: 'user.collective.edit',
         defaultMessage: 'edit profile',
       },
       'user.collective.memberOf.collective.host.title': {
         id: 'user.collective.memberOf.collective.host.title',
-        defaultMessage: "I'm hosting {n, plural, one {this collective} other {these {n} collectives}}",
+        defaultMessage: "I'm hosting {n, plural, one {this Collective} other {these {n} Collectives}}",
       },
       'user.collective.memberOf.organization.admin.title': {
         id: 'user.collective.memberOf.organization.admin.title',
-        defaultMessage: "I'm an administrator of {n, plural, one {this organization} other {these {n} organizations}}",
+        defaultMessage: "I'm an admin of {n, plural, one {this Organization} other {these {n} Organizations}}",
       },
       'user.collective.memberOf.organization.member.title': {
         id: 'user.collective.memberOf.organization.member.title',
-        defaultMessage: "I'm a member of {n, plural, one {this organization} other {these {n} organizations}}",
+        defaultMessage: "I'm a team member of {n, plural, one {this Organization} other {these {n} Organizations}}",
```

Comment:
"team member" is confusing because when editing the organization, it's like collectives: "Core Contributor: Contributor"

Reply:
Ah I see. We said we wanted to call members/admins of organizations "team members". But if that's not possible I guess we could just call them core contributors too...